packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [41.0.0]
+
+### Added
+
+- `TokenListController` now exports the type for `TokensChainsCache` ([#4821](https://github.com/MetaMask/core/pull/4821))
+
```

Comment:
The `TokensChainsCache` type wasn't added as a package-level export, and therefore is unavailable to downstream consumers.

```suggestion
```